src/Grid/grid.jl

```diff
@@ -1450,3 +1605,76 @@ struct SurfaceOrientationInfo
     #flipped::Bool
     #shift_index::Int
 end
+# This one is the same as `SurfaceOrientationInfo` but used in interfaces. It is calculated for the other facet relative to current facet
+struct InterfaceOrientationInfo
+    flipped::Bool
+    transformation::Union{Nothing, AbstractArray}
+end
```

Comment:
This one is really missing a doc string, because it is not obvious what happens here (partially because it is different from the `SurfaceOrientationInfo` struct above). I guess this describes how to transform the face f1 into f2 on the conforming interface described by both faces?

Why `AbstractArray` for a transformation?